settings.gradle.kts

```diff
@@ -204,6 +204,7 @@ include(":instrumentation:cassandra:cassandra-4.4:library")
 include(":instrumentation:cassandra:cassandra-4.4:testing")
 include(":instrumentation:cassandra:cassandra-4-common:testing")
 include(":instrumentation:cdi-testing")
+include(":instrumentation:clickhouse:clickhouse-client:javaagent")
```

Comment:
If there is a module, according to convention, please remove extra `clickhouse` directory. use `clickhouse-client-0.5` directly.